### PR TITLE
parsers: add parser for image builder facts (RHINENG-13943)

### PR DESCRIPTION
**Pull request by croissanne**
### All Pull Requests:

Check all that apply:

* [ ] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [x] No Sensitive Data in this change?
* [ ] Is this PR to correct an issue?
* [x] Is this PR an enhancement?

### Complete Description of Additions/Changes:

*Add your description here*

### CODE REVIEW

File: insights/parsers/image_builder_facts.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+"""
+ImageBuilderFacts - files ``/etc/rhsm/facts/osbuild.facts``
+=========================================================
```
Comment (JoySnow) [inline insights/parsers/image_builder_facts.py:3]: For insights-core doc build, the length of this line shall be align to the line above.

File: insights/parsers/image_builder_facts.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    {
+      "image-builder.insights.compliance-policy-id": "61812cce-c884-4d52-bf05-15b41370db23",
+      "image-builder.insights.compliance-profile-id": "xccdf_org.ssgproject.content_profile_cis",
+      "image-builder.osbuild-composer.api-type": "cloudapi-v2"
+    }
+"""
+
+from insights.specs import Specs
+
+from .. import JSONParser, parser
```
Comment (JoySnow) [inline insights/parsers/image_builder_facts.py:20]: insights-core is promoting the absolute import nowadays, what about change to : 
```suggestion
from insights.core import JSONParser
from insights.core.plugins import parser
```

File: insights/parsers/image_builder_facts.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+
+@parser(Specs.image_builder_facts)
+class ImageBuilderFacts(JSONParser):
+    pass
```
Comment (JoySnow) [inline insights/parsers/image_builder_facts.py:25]: Seeing that you had a doc test `test_image_builder_facts_doc_examples`, could you add a corresponding docstrings test for this parser, an example could be : https://github.com/RedHatInsights/insights-core/blob/master/insights/parsers/bootc.py#L49-L53

**Review by JoySnow — CHANGES_REQUESTED** [review]
Hey @croissanne , thanks for your contribution.  This new parser `ImageBuilderFacts` is all good functional.  Here are some minor change suggestions, please check them inline ...

**Review by xiangce — APPROVED** [review]

**Pull request merged**